.brigade/brigade.ts

```diff
@@ -52,7 +52,6 @@ const publishJob = (event: Event, version: string) => {
   ]
   return job
 }
-jobs[publishJobName] = publishJob
```

Comment:
The jobs map exists to facilitate re-execution of individual check_runs. That is not a concern for this job as this job is only run at release time.